libgalois/include/galois/runtime/Context.h

```diff
@@ -44,10 +42,35 @@ enum ConflictFlag {
   BREAK            = 2
 };
 
-#ifdef GALOIS_USE_LONGJMP_ABORT
-extern thread_local jmp_buf execFrame;
+extern thread_local std::jmp_buf execFrame;
+
+class Lockable;
+
+[[noreturn]] inline void signalConflict(Lockable* = nullptr) {
+#if !(defined(GALOIS_USE_LONGJMP_ABORT) || defined(GALOIS_USE_EXCEPTION_ABORT))
+#error Either GALOIS_USE_LONGJMP_ABORT or GALOIS_USE_EXCEPTION_ABORT must be defined. Check that the Galois config header was included properly.
```

Comment:
nit: Maybe a little more explicit in the developer-facing error message that this is either or but not both?

If we are going to mention the Galois config header, we might as well refer to it by its name `galois/runtime/config.h` so folks know what file this is referring to. I'd also consider removing the second sentence altogether; since from a user's perspective, it is unlikely for them to import `config.h` directly anyway, so the only thing they should be fixing are the multiple defines.

Reply:
Good point. I'll check that it's not both and get rid of the extra message about the config header.